### PR TITLE
New Getting Started Guide

### PR DESCRIPTION
**Pull request by nguyer**
- All of the content under "Getting Started" is completely new.
- This is now completely tailored for someone who is completely new to FireFly and needs a walkthrough on how to get started with it
- Existing "Getting Started Guides" have moved to a new "Tutorials" section

A live preview of these new docs can be viewed at: https://nguyer.github.io/firefly/gettingstarted/gettingstarted.html

### CODE REVIEW

File: docs/gettingstarted/firefly_cli.md [hunk 1]
```diff
@@ -16,22 +16,66 @@ nav_order: 1
...
+## Install the CLI
+
+There are several ways to install the FireFly CLI. The easiest way to get up and running with the FireFly CLI is to download a pre-compiled binary of the latest release.
+
+### Download the package for your OS
+Go to the [latest release page](https://github.com/hyperledger/firefly-cli/releases/latest) and download the package for your OS and CPU architecture.
+
+### Extract the binary and move it to `/usr/bin/local`
+
+Assuming you downloaded the package from GitHub into you `Downloads` directory, run the following command:
```
Comment (nickgaski) [inline docs/gettingstarted/firefly_cli.md:48]: into your

File: docs/gettingstarted/setup_env.md [hunk 2]
```diff
@@ -16,62 +16,128 @@ nav_order: 2
...
+A FireFly stack is a collection of Supernodes with networking and configuration that are designed to work together on a single development machine. A stack has multiple members (also referred to organizations). Each member has their own Supernode within the stack. This allows developers to build and test data flows with a mix of public and private data between various parties, all within a single development environment.
 
-## The runtimes you will get
+![FireFly Stack](../images/firefly_stack.svg)
 
-The CLI includes a number of options (viewable via `ff help init`) to control the runtimes that will be created
-within your stack. By default, it will include the following:
+The stack also contains an instance of the FireFly Sandbox for each member. This is an example of an end-user application that uses FireFly's API. It has a backend and a frontend which are designed to walk developers through the features of FireFly, and provide code snippets and as examples of how to build those features into their own application. The next section in this guide will walk you through using the Sandbox.
```
Comment (nickgaski) [inline docs/gettingstarted/setup_env.md:36]: , and provide codes snippets as examples of how to build...

File: docs/gettingstarted/setup_env.md [hunk 3]
```diff
@@ -16,62 +16,128 @@ nav_order: 2
...
+```
+stack name: dev
+```
+
+Chose the number of members for your stack. For this guide, we should pick `3` members, so we can try out both public and private messaging use cases:
+```
+number of members: 3
+```
+
+![ff start](../images/ff_start.gif)
```
Comment (nickgaski) [inline docs/gettingstarted/setup_env.md:63]: 🔥

File: docs/gettingstarted/sandbox.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+The Sandbox is split up into three columns:
+
+### Left column: Prepare your request
+On the left-hand side of the page, you can fill out simple form fields to construct messages and more. Some tabs have more types of requests on them in sections that can be expanded or collapsed. Across the top of this column there are three tabs that switch between the three main sets of functionality in the Sandbox. The next three sections of this guide will walk you through each one of these.
+
+The first tab we will explore is the **MESSAGING** tab. This is where we can send broadcasts and private messages.
+
+### Middle column: Preview server code and see response
+As you type in the form on the left side of the page, you may notice that the source code in the top middle of the page updates automatically. If you were building a backend app, this is an example of code that your app could use to call the FireFly SDK. The middle column also contains the button to actually send the request.
```
Comment (nickgaski) [inline docs/gettingstarted/sandbox.md:83]: The middle column also contains a **RUN** button to actually send the request.

File: docs/gettingstarted/sandbox.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+The Tokens tab is where you can create token pools, and mint, burn, or transfer tokens. This works with both fungible and non-fungible tokens (NFTs). For more details, please see the tutorial on [Minting some tokens](../tutorials/mint_tokens.md).
+
+### Things to try out
+
+- Create a fungible token pool and mint some tokens and view your balance in the FireFly Explorer
+- Transfer some amount of those tokens to another member and view the transfer transaction in the FireFly Explorer
+- Burn some amount of tokens and view the transaction and updated balances in the FireFly Explorer
+- Create a non-fungible token pool and mint some NFTs
+- Transfer an NFT to another member and verify the account balances in the FireFly Explorer
+
```
Comment (nickgaski) [inline docs/gettingstarted/sandbox.md:111]: maybe also add a suggestion to create a tx failure like you showcased in the demo.  

"attempt to transfer an amount that exceeds your balance, and investigate the associated event"

File: docs/gettingstarted/sandbox.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+The Contracts section of the Sandbox lets you interact with custom smart contracts, right from your web browser! The Sandbox also provides some helpful tips on deploying your smart contract to the blockchain. For more details, please see the tutorial on [Working with custom smart contracts](../tutorials/custom_contracts.md).
+
+![Sandbox Tokens](../images/sandbox/sandbox_contracts_api.png)
+
+![Sandbox Tokens](../images/sandbox/sandbox_api_swagger.png)
+
+### Things to try out
+
+- Create a contract interface and API then view the Swagger UI for your new API
```
Comment (nickgaski) [inline docs/gettingstarted/sandbox.md:124]: and API, then view...

**Review by nickgaski — CHANGES_REQUESTED** [review]
great stuff @nguyer - couple minor suggestions

File: docs/gettingstarted/sandbox.md (hunk 5 above)
Comment (nguyer) [reply docs/gettingstarted/sandbox.md:111]: I would suggest that... except it kind of breaks the sandbox right now

**Review by nickgaski — APPROVED** [review]

**Pull request merged**